### PR TITLE
Fix up errors

### PR DESCRIPTION
**Pull request by rmccue**
* There was an unused function for sending traces via HTTP. We don't use this anywhere, and it references a now non-existent function, so removed that.
* Fixed the end time for the main trace, which was a copypasta error
* Fixed a Joe spelling mistake, classic

### CODE REVIEW

File: inc/namespace.php [hunk 1]
```diff
@@ -32,10 +30,10 @@ function bootstrap() {
 		register_shutdown_function( __NAMESPACE__ . '\\on_shutdown' );
 	}
 
-	$current_errror_handler = set_error_handler( function () use ( &$current_errror_handler ) { // @codingStandardsIgnoreLine
+	$current_error_handler = set_error_handler( function () use ( &$current_error_handler ) { // @codingStandardsIgnoreLine
```
Comment (roborourke) [inline inc/namespace.php]: Unrelated to this PR but is the ampersand still needed? Think this will break in PHP 7.3

File: inc/namespace.php [hunk 2]
```diff
@@ -443,7 +422,7 @@ function get_xhprof_trace() : array {
 			'value'      => 1,
 			'children'   => [],
 			'start_time' => $hm_platform_xray_start_time,
-			'end_time'   => $hm_platform_xray_start_time,
+			'end_time'   => $end_time,
```
Comment (roborourke) [inline inc/namespace.php]: clbuttic

**Comment by roborourke** [comment]
Actually should confirm that `hm-platform` doesn't use that function. Pretty sure Altis has it's own version

File: inc/namespace.php (hunk 1 above)
Comment (rmccue) [reply inc/namespace.php]: Yes; pass-by-ref is required in `use` statements (unlike function arguments) as it can alter the parent scope's variable.

**Comment by rmccue** [comment]
Sourcegraph shows the only existence of that function at all is in the declaration (it's also been copied into an old project back in the pre-hm-platform days).

The function existed to synchronously send data to X-Ray, but we switched all usage to send asynchronously via the system daemon instead.

**Pull request merged**